docker-compose-minimal.yml

```diff
@@ -0,0 +1,27 @@
+version: '3.8'
```

Comment:
this doesn't seem related to the story and isn't reference anywhere?

Reply:
I use this to spin up a local development environment with minimal local docker images (postgres and redis). Happy to put a line in the readme to reference it 😄

Reply:
if you think its worth adding then it's probably worth adding to the README